lib/commands/execute.js

```diff
@@ -37,7 +37,9 @@ extensions.executeMobile = async function (mobileCommand, opts={}) {
     tap: async (x) => await this.mobileTap(x),
     dragFromToForDuration: async (x) => await this.mobileDragFromToForDuration(x),
     selectPickerWheelValue: async (x) => await this.mobileSelectPickerWheelValue(x),
-    alert: async (x) => await this.mobileHandleAlert(x)
+    alert: async (x) => await this.mobileHandleAlert(x),
+    setPasteboard: async (x) => await this.mobileSetPasteboard(x),
+    getPasteboard: async (x) => await this.mobileGetPasteboard(x),
```

Comment:
strikes me incidentally that this could be made a bit less redundant:

```js
const mobileCommandsMapping = {
  scroll: 'mobileScroll',
  pinch: 'mobilePinch',
  ...
};
...
await this[mobileCommandsMapping[mobileCommand]](opts);
```

(gets rid of all the unnecessary lambdas... i know there's a wrinkle with `swipe` but that could be made into its own method `mobileSwipe` which just calls `mobileScroll`)